ACPs/77-reinventing-subnets/README.md

```diff
@@ -268,70 +275,71 @@ type RegisterSubnetValidatorTx struct {
     // below in `Message` for this validator.
     // Note: We do not enforce that the BLS key is unique across all validators.
     //       This means that validators can share a key if they so choose.
-    //       However, a NodeID does uniquely map to a BLS key
+    //       However, a NodeID + L1 does uniquely map to a BLS key
     Signer [96]byte `json:"signer"`
-    // A RegisterSubnetValidatorMessage payload
+    // A RegisterL1ValidatorMessage payload
     Message warp.Message `json:"message"`
 }
 ```
 
-The `validationID` of validators added via `RegisterSubnetValidatorTx` is defined as the SHA256 hash of the `Payload` of the `AddressedCall` in `Message`.
+The `validationID` of validators added via `RegisterL1ValidatorTx` is defined as the SHA256 hash of the `Payload` of the `AddressedCall` in `Message`.
 
-When a `RegisterSubnetValidatorTx` is accepted on the P-Chain, the Subnet Validator is added to the Subnet's validator set. A `minNonce` field corresponding to the `validationID` will be stored on addition to the validator set (initially set to `0`). This field will be used when validating the `SetSubnetValidatorWeightTx` defined below.
+When a `RegisterL1ValidatorTx` is accepted on the P-Chain, the validator is added to the L1's validator set. A `minNonce` field corresponding to the `validationID` will be stored on addition to the validator set (initially set to `0`). This field will be used when validating the `SetL1ValidatorWeightTx` defined below.
 
-This `validationID` will be used for replay protection. Used `validationID`s will be stored on the P-Chain. If a `RegisterSubnetValidatorTx`'s `validationID` has already been used, the transaction will be considered invalid. To prevent storing an unbounded number of `validationID`s, the `expiry` of the `RegisterSubnetValidatorMessage` is required to be no more than 48 hours in the future of the time the transaction is issued on the P-Chain. Any `validationIDs` corresponding to an expired timestamp can be flushed from the P-Chain's state.
+This `validationID` will be used for replay protection. Used `validationID`s will be stored on the P-Chain. If a `RegisterL1ValidatorTx`'s `validationID` has already been used, the transaction will be considered invalid. To prevent storing an unbounded number of `validationID`s, the `expiry` of the `RegisterL1ValidatorMessage` is required to be no more than 48 hours in the future of the time the transaction is issued on the P-Chain. Any `validationIDs` corresponding to an expired timestamp can be flushed from the P-Chain's state.
 
-Subnets are responsible for defining the procedure on how to retrieve the above information from prospective validators.
+L1s are responsible for defining the procedure on how to retrieve the above information from prospective validators.
 
-An EVM Subnet may choose to implement this step like so:
+A Subnet-EVM based L1 may choose to implement this step like so:
```

Comment:
It feels strange to me to call out Subnet-EVM in particular. This is generic for any EVM Subnet- what do you think about reverting?

```suggestion
An EVM Subnet may choose to implement this step like so:
```

Reply:
the language should apply only to L1s, as Subnets dont define their validator control logic, the pchain does.

Reply:
https://github.com/avalanche-foundation/ACPs/pull/154/files#r1816822953